src/license/license.proto

```diff
@@ -65,6 +72,9 @@ message ClusterStatus {
 message UpdateClusterRequest {
   string id = 1;
   string address = 2;
+  string user_address = 3;
+  string cluster_deployment_id = 4;
+  bool enterprise_server = 5;
```

Comment:
Since `enterprise_server` is a boolean field in an update call, we always interpret a value even if the caller didn't explicitly set it. 

@actgardner let me know if you have any concern here. if it's not a useful enough field in the context of the Update RPC, we could also cut it out.

Reply:
Yeah let's not support updating it - in the worst case a user can delete the cluster and recreate it.